buffer/buffer.php

```diff
@@ -16,50 +16,56 @@
 use Friendica\Database\DBA;
 use Friendica\Model\ItemContent;
 
-function buffer_install() {
+function buffer_install()
+{
 	Addon::registerHook('post_local',           'addon/buffer/buffer.php', 'buffer_post_local');
 	Addon::registerHook('notifier_normal',      'addon/buffer/buffer.php', 'buffer_send');
 	Addon::registerHook('jot_networks',         'addon/buffer/buffer.php', 'buffer_jot_nets');
 	Addon::registerHook('connector_settings',      'addon/buffer/buffer.php', 'buffer_settings');
 	Addon::registerHook('connector_settings_post', 'addon/buffer/buffer.php', 'buffer_settings_post');
 }
 
-function buffer_uninstall() {
-	Addon::unregisterHook('post_local',       'addon/buffer/buffer.php', 'buffer_post_local');
-	Addon::unregisterHook('notifier_normal',  'addon/buffer/buffer.php', 'buffer_send');
-	Addon::unregisterHook('jot_networks',     'addon/buffer/buffer.php', 'buffer_jot_nets');
+function buffer_uninstall()
+{
+	Addon::unregisterHook('post_local',              'addon/buffer/buffer.php', 'buffer_post_local');
+	Addon::unregisterHook('notifier_normal',         'addon/buffer/buffer.php', 'buffer_send');
+	Addon::unregisterHook('jot_networks',            'addon/buffer/buffer.php', 'buffer_jot_nets');
 	Addon::unregisterHook('connector_settings',      'addon/buffer/buffer.php', 'buffer_settings');
 	Addon::unregisterHook('connector_settings_post', 'addon/buffer/buffer.php', 'buffer_settings_post');
 }
 
-function buffer_module() {}
-
-function buffer_content(&$a) {
+function buffer_module()
```

Comment:
For those cases, I would really appreciate the one-liner, even though the braces might have to go to the next lines according to the standard.

Reply:
I've never liked this way of signalling the addon provides a module. I feel like we should check for the existence of either `addon_content()` or `addon_post()`.

Reply:
And continue rewriting it to a object-oriented way. Like with filter chains?

Reply:
Ideally, addons would provide a `Module` class and hook into the routing mechanism to add their custom routes. Unfortunately at the moment we only have the former, with an implicit router that assumes that all modules are in `src/Module`.